### PR TITLE
Add protobuf to Hll

### PR DESCRIPTION
**Pull request by timkaye11**
- add protobuf serialization to Hll 
 - tested protobuf roundtrip for Hll

### CODE REVIEW

**Comment by araddon** [comment]
lgtm :+1:

File: hll.proto [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package hll;
+
+message hllPb {
```
Comment (mdmarek) [inline hll.proto]: Do we usually have these match the struct name, like `HllPb` ?

Comment (timkaye11) [inline hll.proto]: nope. I didn't have a particular name in mind so I went with `HllPb` . any other ideas?

**Comment by mdmarek** [comment]
:+1:

**Pull request merged**